packages/ag-charts-community/src/chart/series/cartesian/barSeriesModule.ts

```diff
@@ -37,17 +32,17 @@ export const BarSeriesModule: SeriesModule<'bar'> = {
     themeTemplate: {
         __extends__: EXTENDS_SERIES_DEFAULTS,
         fillOpacity: 1,
-        strokeOpacity: 1,
+        strokeOpacity: 0,
         strokeWidth: 1,
         lineDash: [0],
         lineDashOffset: 0,
         label: {
             enabled: false,
             fontStyle: undefined,
-            fontWeight: undefined,
+            fontWeight: 'bold',
             fontSize: 12,
             fontFamily: DEFAULT_FONT_FAMILY,
-            color: DEFAULT_LABEL_COLOUR,
+            color: 'white',
```

Comment:
Does this work for dark theme?

Reply:
I think for now it will, as it's only used for inside the series items which will never be white

Reply:
Shouldn't this be the new symbol you've introduced? (`DEFAULT_INSIDE_SERIES_LABEL_COLOUR`)